public/src/managers/CourseManager.ts

```diff
@@ -362,6 +388,27 @@ export class CourseManager {
         }
     }
 
+    /**
+     * Add IStudentSubmissions to an IUserCourse
+     * @param group The group
+     * @param groupCourse The group course
+     */
+    private async fillLinksGroup(group: ICourseGroup, groupCourse: IGroupCourse): Promise<void> {
+        if (!groupCourse.link) {
+            return;
+        }
+        const assigns = await this.getAssignments(groupCourse.course.id);
+        if (assigns.length > 0) {
+            const submissions = MapHelper.toArray(
+                await this.courseProvider.getAllGroupLabInfos(groupCourse.course.id, group.id));
+
+            for (const a of assigns) {
+                const temp = submissions.find((sub) => sub.assignmentid === a.id);
```

Comment:
`temp` --> `submission` ?? (I think that is what it is.)

Reply:
`assignmentid` --> `assignmentID` (see other comment below about ID vs Id vs id)

(Not sure if `a.id` should be replaced with `a.ID`, but I think it should be ok in Typescript, since capitalization won't export the field, unless I'm mistaken.)